react-website/src/pages/Table.js

```diff
@@ -42,22 +52,24 @@ function Table() {
         })
     // Keep the empty array.
     }, []);
-    
+
+    const ObjectId = '_id';
```

Comment:
This line of code isn't being used anymore. Can remove.

Reply:
fixed